db/migrate/20231003182930_add_species2_pets.rb

```diff
@@ -0,0 +1,5 @@
+class AddSpecies2Pets < ActiveRecord::Migration[7.0]
+  def change
+    add_column :pets, :species, :integer
```

Comment:
```suggestion
    add_column :pets, :species, :integer, null: false
```
Adding this `null: false` prevents at the database level from allowing rows without the species defined. Helps keep data more accurate in case of issues. This comes especially helpful if validations get skipped

Reply:
@jadekstewart3 in case you missed this comment :)

Reply:
@jadekstewart3 I think the `schema.rb` file was not updated with `null: false`. Easiest way to regenerate it is to run `rails db:rollback` and `rails db:migrate` :)

Reply:
@Ptrboro Oh heck! Thank you!